src/storage/hummock_test/src/compactor_tests.rs

```diff
@@ -1308,4 +1317,252 @@ pub(crate) mod tests {
         assert_eq!(1, output_level_info.table_infos.len());
         assert_eq!(252, output_level_info.table_infos[0].total_key_count);
     }
+
+    type KeyValue = (FullKey<Vec<u8>>, HummockValue<Vec<u8>>);
+
+    async fn test_fast_compact_impl(
+        data1: Vec<KeyValue>,
+        data2: Vec<KeyValue>,
+        data3: Vec<KeyValue>,
+    ) {
+        let (env, hummock_manager_ref, _cluster_manager_ref, worker_node) =
+            setup_compute_env(8080).await;
+        let hummock_meta_client: Arc<dyn HummockMetaClient> = Arc::new(MockHummockMetaClient::new(
+            hummock_manager_ref.clone(),
+            worker_node.id,
+        ));
+        let existing_table_id: u32 = 1;
+        let storage = get_hummock_storage(
+            hummock_meta_client.clone(),
+            get_notification_client_for_test(env, hummock_manager_ref.clone(), worker_node.clone()),
+            &hummock_manager_ref,
+            TableId::from(existing_table_id),
+        )
+        .await;
+        hummock_manager_ref.get_new_sst_ids(10).await.unwrap();
+        let compact_ctx = prepare_compactor_and_filter(&storage, existing_table_id);
+
+        let sstable_store = compact_ctx.sstable_store.clone();
+        let capacity = 4 * 1024 * 1024;
+        let mut options = SstableBuilderOptions {
+            capacity,
+            block_capacity: 2048,
+            restart_interval: 16,
+            bloom_false_positive: 0.1,
+            ..Default::default()
+        };
+        let sst1 = gen_test_sstable_info(options.clone(), 1, data1, sstable_store.clone()).await;
+        let sst2 = gen_test_sstable_info(options.clone(), 2, data2, sstable_store.clone()).await;
+        options.compression_algorithm = CompressionAlgorithm::Lz4;
+        let sst3 = gen_test_sstable_info(options, 3, data3, sstable_store.clone()).await;
+        let read_options = Arc::new(SstableIteratorReadOptions::default());
+
+        let task = CompactTask {
+            input_ssts: vec![
+                InputLevel {
+                    level_idx: 5,
+                    level_type: 1,
+                    table_infos: vec![sst1.clone()],
```

Comment:
We only have one sst in this level. Will this test cover the branch that we need to call `next_sstable` in fast compaction runner?

Reply:
The next level has two files, I think it is no effect. But I have change it in newest commit